01_data-model-and-serialized-rep.adoc

```diff
@@ -1571,12 +1567,22 @@ can have significant performance consequences since the server may need
 to read and serialize all of the data for all of the variables mentioned
 in the DMR even though that data is not transmitted to the client.
 
+For a dmr response the checksum is made visible to the client by the addition
```

Comment:
I think it important to overstate things, for example:
`For a dmr-only response, when requested the checksum is made ...`

Reply:
I tried to clarify per your suggestions, please re-read this section.

Reply:
Also - I think the text box above that shows the text is marked **_Outdated_** and is not up to date with the changes I have made (Thanks GitHub!) Maybe a better tactic would be to get the branch, pull the changes and look at this section locally.

Reply:
That is frustrating - I will check locally